docs/topics/proc_deploying-the-http-api-booster-using-the-oc-cli-client.adoc

```diff
@@ -80,9 +81,9 @@ The number in the middle will increase with each new build.
 The letters at the end are generated when the pod is created.
 
 
-. Once your booster is deployed and started, determine its route.
+. Once your example is deployed and started, determine its route.
```

Comment:
- example is deployed--> example application is deployed

- Once--> After (Once means one time. The IBM Style Guide prohibits its usage to mean "after" or "when")

Reply:
Good catch. I updated this for the similar modules as well.